### PR TITLE
Add policy based routing test

### PR DESCRIPTION
**Pull request by glazychev-art**
Issue: https://github.com/networkservicemesh/api/issues/119

Signed-off-by: Artem Glazychev <artem.glazychev@xored.com>

### CODE REVIEW

**Comment by glazychev-art** [comment]
@edwarnicke @zolug @tedlean
Could you please take a look at this example? It is related to Policy Based Routing issue.
Is this what we want to see?

File: examples/features/policy-based-routing/config-file-nse.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: policies-config-file
+data:
+  config.yaml: |
+    - from: 172.16.2.201/24
+      proto: 6
```
Comment (denis-tingaikin) [inline examples/features/policy-based-routing/config-file-nse.yaml:9]: This field is super hard to understand. I think we should consider improving clarity.

File: examples/features/policy-based-routing/config-file-nse.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+      port: 6666
+      routes:
+        - prefix: 172.16.3.0/24
+          nexthop: 172.16.2.200
+    - proto: 17
+      port: 6666
+    - from: 2004::3/120
+      proto: 17
```
Comment (denis-tingaikin) [inline examples/features/policy-based-routing/config-file-nse.yaml:17]: This field is super hard to understand. I think we should consider improving clarity.

File: examples/features/policy-based-routing/config-file-nse.yaml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  name: policies-config-file
+data:
+  config.yaml: |
+    - from: 172.16.2.201/24
+      proto: 6
+      port: 6666
+      routes:
+        - prefix: 172.16.3.0/24
+          nexthop: 172.16.2.200
+    - proto: 17
```
Comment (denis-tingaikin) [inline examples/features/policy-based-routing/config-file-nse.yaml:14]: This field is super hard to understand. I think we should consider improving clarity.

**Review by denis-tingaikin — COMMENTED** [review]
@glazychev-art Is it possible to replace `proto: ${number}` to something human readable?

**Comment by denis-tingaikin** [comment]
OK, I see that protocol numbers are known https://en.wikipedia.org/wiki/List_of_IP_protocol_numbers

So merging.


@edwarnicke Let us know if you will have any questions or comments on this example.

**Pull request merged**